ush/parsing_namelists_FV3.sh

```diff
@@ -19,6 +19,11 @@ DIAG_TABLE_APPEND=${DIAG_TABLE_APPEND:-${PARMgfs}/ufs/fv3/diag_table_aod}
 DATA_TABLE=${DATA_TABLE:-${PARMgfs}/ufs/MOM6_data_table.IN}
 FIELD_TABLE=${FIELD_TABLE:-${PARMgfs}/ufs/fv3/field_table}
 
+# set cdmbgwd
+if [[ ${do_gsl_drag_ls_bl} == ".true." ]]; then
+ export cdmbgwd=${cdmbgwd_gsl}
+fi
```

Comment:
Should also edit `parm/config/gfs/config.fcst` to remove this block from there.  Its redundant.
https://github.com/NOAA-EMC/global-workflow/pull/2392/files#diff-e3a649dff56be3c9659fdfb052461b30ef47d5e45c2a8a5bafb4c62c30581967R118-R120